.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 2018e667a6a36ee3fbfa8041cd36512f92f60d49  # frozen: 22.8.0
+    hooks:
+    - id: black
+      language_version: python3.8
```

Comment:
What do you think of leaving language_version off ? I don't happen to have python3.8 available on my machine so I suspect others might not either

Reply:
Yes, sounds like a great idea.